AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -829,15 +841,7 @@ - (CGSize)viewportSizeForRangeController:(ASRangeController *)rangeController
 
 - (ASInterfaceState)interfaceStateForRangeController:(ASRangeController *)rangeController
 {
-  ASCollectionNode *collectionNode = self.collectionNode;
-  if (collectionNode) {
-    return self.collectionNode.interfaceState;
-  } else {
-    // Until we can always create an associated ASCollectionNode without a retain cycle,
-    // we might be on our own to try to guess if we're visible.  The node normally
-    // handles this even if it is the root / directly added to the view hierarchy.
-    return (self.window != nil ? ASInterfaceStateVisible : ASInterfaceStateNone);
-  }
+  return self.interfaceState;
```

Comment:
We could simply leave this method alone, Or we could keep the newly defined interface state method and simply not expose that API. We could also put that API in the +Beta header